ppl/zqd/core.go

```diff
@@ -127,11 +127,11 @@ func NewCore(ctx context.Context, conf Config) (*Core, error) {
 
 	var startFields []zap.Field
 	switch personality {
-	case "all", "apiserver":
+	case "all", "apiserver", "root":
 		if err := c.addAPIServerRoutes(ctx, conf); err != nil {
 			return nil, err
 		}
-		if personality == "all" {
+		if personality == "all" || personality == "root" {
```

Comment:
Should we drop the use of personality "all" then?

Reply:
I think the personality "root" will be temporary, and we will remove it after we start using personality "apiserver" in our deploys. I think we should keep personality "all".